package/suse-migration-services-spec-template

```diff
@@ -38,14 +38,23 @@ Requires:         ca-certificates
 Requires:         dialog
 Requires:         rsync
 Requires:         SUSEConnect >= 0.3.20
-Requires:         suse-migration-pre-checks
 Requires(preun):  systemd
 Requires(postun): systemd
 BuildArch:        noarch
 
 %description
 Systemd services to prepare and run a distribution migration process.
 
+%package          -n suse-migration-pre-checks
+Summary:          The pre-checks code used with suse-migration-services
+Group:            System/Management
+BuildRoot:        %{_tmppath}/%{name}-%{version}-build
+Requires:         python3-Cerberus
+Conflicts:        suse-migration-sle15-activation < 2.0.33
```

Comment:
It does conflict with the old activation as suse-migration-sle15-activation-2.0.32 installs the python modules that are part of suse-migration-services.  It does this since the pre-checks were originally part suse-migration-sle15-activation. I should have removed the python modules from suse-migration-sle15-activation when the suse-migration-pre-checks was first created.  I originally only removed the suse-migration-pre-checks console script and not all of other python stuff pulled in from suse-migration-services.  The new suse-migration-sle15-activation-2.0.33 does not include this any more (Ideally suse-migration-sle15-activation could be done differently now as it only contains the grub stuff and using the sdist is not needed.) Without this "Conflicts: suse-migration-sle15-activation < 2.0.33" you get a zypper conflict message if the customer would somehow upgrade from 2.0.32 to 2.0.33 before doing he migration.

The error without "Conflicts: suse-migration-sle15-activation < 2.0.33"
````
File /usr/lib/python3.4/site-packages/suse_migration_services/prechecks/pre_checks.py
  from install of
     suse-migration-pre-checks-2.0.33-1.83.1.noarch (Plain RPM files cache)
  conflicts with file from package
     suse-migration-sle15-activation-2.0.32-6.27.1.noarch (@System)

File conflicts happen when two packages attempt to install files with the same name but different contents. If you continue, conflicting files will be replaced losing the previous content.
Continue? [yes/no] (no): no

```
The message with "Conflicts: suse-migration-sle15-activation < 2.0.33" added in  (you get the expected if they try to upgrade)

```
Problem: suse-migration-pre-checks-2.0.33-1.84.1.noarch conflicts with suse-migration-sle15-activation < 2.0.33 provided by suse-migration-sle15-activation-2.0.32-6.27.1.noarch
 Solution 1: deinstallation of suse-migration-sle15-activation-2.0.32-6.27.1.noarch
 Solution 2: do not install suse-migration-pre-checks-2.0.33-1.84.1.noarch
```
If you think this is a corner case we do not need to consider,I can remove it. I do not think there is any harm in leaving it in as it provides a safety net.